.gitmodules

```diff
@@ -277,3 +277,6 @@
 [submodule "ext/sdk/resources/sdk-root/fxcode"]
 	path = ext/sdk/resources/sdk-root/fxcode
 	url = https://github.com/citizenfx/fxcode.git
+[submodule "vendor/rnnoise"]
+	path = vendor/rnnoise
+	url = https://github.com/almoghamdani/rnnoise-cmake
```

Comment:
why weird fork

Reply:
it's just rnnoise for cmake, because by default it's supports only make(

Reply:
so, i can fork it, if it changes something

Reply:
i change it for my fork from original repository, now it's should be fine :)

Reply:
>it's just rnnoise for cmake, because by default it's supports only make(

the original repo has the same layout, doesn't it? you're not even using either build system here. why?

Reply:
so, now i deleted useless cmake, but by default rnnoise can't compiled on windows, my fork have some changes to fix it